postgres/docker-compose.yml

```diff
@@ -3,17 +3,20 @@ version: '2'
 services:
     postgres:
         image: codefordc2/postgres
```

Comment:
Remind me to publish a new version of the image after we merge this.

Reply:
Yeah, there is currently not a /postgres image. The documentation says "replaces codefordc2/postgres with the name of your image" assuming people will follow the pre-populated image creation path. 

It would be good to replace the housing-insights-postgres image in the top level compose file with a generic one - I kept that one in there b/c it provides an empty database version that is useful for anyone that adds their data in via an ingestion process.